noncommands/quotes.py

```diff
@@ -2,52 +2,50 @@
 import random
 import yaml
 import os
+import re
 
 if "CompsciBot" not in str(os.getcwd()):
     os.chdir("./CompsciBot")
 with open("config.yaml") as file:
     config = yaml.load(file, Loader=yaml.FullLoader)
 
+
 class Quotes:
     def __init__(self, bot):
-        self.bot=bot
+        self.bot = bot
+
     async def dailyQuote(self):
-        c=self.bot.get_channel(707293854507991172)
-        dailyquote=await Quotes.quote(self.bot,"")
+        c = self.bot.get_channel(707293854507991172)
+        dailyquote = await Quotes.quote(self.bot, "")
         await c.send("Daily Quote:\n"+dailyquote)
 
-    async def quote(self,keywords):
+    async def quote(self, keywords):
 
         f = open("resources/quotes.json", encoding="utf-8")
         json_data = json.load(f)
         quotes = list(json_data['teacherQuotes'])
         matches = []
 
         if keywords:
-            random_quote="Sorry! You made a bad search"
+            random_quote = "Sorry! You made a bad search"
             for line in quotes:
-                if keywords in line:
+                if re.search(keywords, line, re.IGNORECASE):
```

Comment:
Are we sure giving users regex is a good idea?

This wasn't mentioned anywhere in the PR description is this intended behavior?